x-pack/test/stack_functional_integration/configs/config.stack_functional_integration_base_ie.js

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { resolve } from 'path';
+import buildState from './build_state';
+import { ToolingLog } from '@kbn/dev-utils';
+import chalk from 'chalk';
+
+const reportName = 'Stack Functional Integration Tests';
+const testsFolder = '../test/functional/apps';
+const stateFilePath = '../../../../../integration-test/qa/envvars.sh';
+const prepend = (testFile) => require.resolve(`${testsFolder}/${testFile}`);
+const log = new ToolingLog({
+  level: 'info',
+  writeTo: process.stdout,
+});
+
+export default async ({ readConfigFile }) => {
+  const defaultConfigs = await readConfigFile(require.resolve('../../functional/config'));
+  const { servers } = defaultConfigs.getAll();
+  const { tests, ...provisionedConfigs } = buildState(resolve(__dirname, stateFilePath));
+
+  return {
+    ...defaultConfigs.getAll(),
+    junit: {
+      reportName: `${reportName} - ${provisionedConfigs.VM}`,
+    },
+    browser: {
+      type: 'ie',
+    },
+    servers,
+    security: { disableTestUser: true },
```

Comment:
This whole file should just extend the base config file with the couple of changes needed for IE.
the `browser` and the `security` config parts at least.

Reply:
+1, this file has just 2 different lines from base config